src/containers/enrollment/EnrollmentLink.js

```diff
@@ -0,0 +1,42 @@
+import { faCopy } from '@fortawesome/pro-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// @flow
+import {
+  Button,
+  Card,
+  CardHeader,
+  CardSegment,
+} from 'lattice-ui-kit';
+
+import copyToClipboard from '../../utils/copyToClipboard';
+
+const EnrollmentLink = () => {
+  const enrollmentLinkQueryParams = window.location.hash.split('?')[1];
+  const enrollmentLink = `http://openlattice.com/login?${enrollmentLinkQueryParams}`;
```

Comment:
The link is missing `/chronicle` in the path.

```suggestion
  const enrollmentLink = `http://openlattice.com/chronicle/login?${enrollmentLinkQueryParams}`;
```

Reply:
oops, thank you